app/views/simulator/edit.html.erb

```diff
@@ -620,10 +620,12 @@
     $('.dropdown.open').removeClass('open');
     $(this).parent('.dropdown').addClass('open');
   });
+
+
 </script>
 
 <link rel="stylesheet" type="text/css" href="/css/restrictedElements.css">
-
+<link rel="stylesheet" type="text/css" href="../../../simulator/src/css/main.stylesheet.css">
```

Comment:
Reason for importing this ?

Reply:
I have removed that line now,the reason why included because the css changes aren't getting reflected,but now it's perfect.